viewing_party/party.py

```diff
@@ -1,23 +1,147 @@
 # ------------- WAVE 1 --------------------
-
+from ast import AnnAssign
+from curses import use_default_colors
+from operator import ne
+import re
+import copy
+# from tkinter import N
 def create_movie(title, genre, rating):
-    pass
+    dic = {}
+    dic["title"] = title
+    dic["genre"] = genre
+    dic["rating"] = rating
+    if dic["title"] == None or dic["genre"] == None or dic["rating"] == None:

```

Comment:
This works, though it is better practice to use `is` to check for None values. [This article](https://www.pythontutorial.net/advanced-python/python-none/) gives a good explanation why! It gets into how python objects' equality is defined, something we'll cover in OOP.